packages/core/src/schema/models/commits.ts

```diff
@@ -44,7 +44,10 @@ export const commits = latitudeSchema.table(
 
 export const commitRelations = relations(commits, ({ one, many }) => ({
   snapshots: many(documentSnapshots, { relationName: 'snapshots' }),
-  workspace: one(workspaces),
+  workspace: one(workspaces, {
+    fields: [commits.workspaceId],
+    references: [workspaces.id],
+  }),
```

Comment:
this is gonna change so i'd probably wouldn't touch it (andres is moving tenancy of commits to projects)